lib/src/charts/widgets/bar_chart.dart

```diff
@@ -27,6 +27,12 @@ class BarChart extends StatefulWidget {
   /// Defaults to true.
   final bool autoAnimate;
 
+  /// The Duration for which the chart should animate
+  final Duration? animDuration;
```

Comment:
Use full names please. 
```suggestion
  final Duration? animationDuration;
```